python/lsst/ap/verify/ap_verify.py

```diff
@@ -152,8 +157,11 @@ def runApVerify(cmdLine=None):
     ingestDataset(args.dataset, workspace)
 
     log.info('Running pipeline...')
-    expandedDataIds = runApPipe(workspace, args)
-    computeMetrics(workspace, expandedDataIds, args)
+    apPipeResults = runApPipe(workspace, args)
+    computeMetrics(workspace, apPipeResults.parsedCmd.id, args)
+
+    # ApPipeTaskRunner does not override default results handling, exitStatus always defined
+    return sum(((res.exitStatus != 0) for res in apPipeResults.resultList))
```

Comment:
This code is unsafe, because lsst/afw#97 allows `resultList` to be `None` (not clear if its elements can be `None` or not).

Reply:
Hmm, oh dear, good real-time catch. Could you fix this with a try/except for the return value and have it return something else if summing the exitStatuses doesn't work?